SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Ingestion/Models/StartServiceLog.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+
```

Comment:
Do you mean [this identifier](https://github.com/Microsoft/mobile-center-sdk-dotnet/pull/157/files/#diff-9dc6c835d2b9e5798e8d1e826c2d67d6R9)?